### ISSUE TITLE
[RL4j]Output from network is not a probability distribution

### ISSUE THREAD

**Issue by RuralHunter** [deeplearning4j/deeplearning4j#9066]
#### Issue Description
There is an exception sometimes when calling ACPolicy.nextAction(), such as this:
```
Exception in thread "Thread-10" java.lang.RuntimeException: Output from network is not a probability distribution: [[    0.4998,    0.5002]]
	at org.deeplearning4j.rl4j.policy.ACPolicy.nextAction(ACPolicy.java:88)
	at org.deeplearning4j.rl4j.policy.ACPolicy.nextAction(ACPolicy.java:71)
	at org.deeplearning4j.rl4j.policy.ACPolicy.nextAction(ACPolicy.java:38)
	at org.deeplearning4j.rl4j.learning.async.AsyncThreadDiscrete.trainSubEpoch(AsyncThreadDiscrete.java:104)
	at org.deeplearning4j.rl4j.learning.async.AsyncThread.handleTraining(AsyncThread.java:192)
	at org.deeplearning4j.rl4j.learning.async.AsyncThread.run(AsyncThread.java:168)
```
[[    0.4998,    0.5002]] clearly is a probability distribution. I checked the [source code](https://github.com/eclipse/deeplearning4j/blob/6b736eae3333a94c1ef22ae5e53b1139434bb4b5/rl4j/rl4j-core/src/main/java/org/deeplearning4j/rl4j/policy/ACPolicy.java#L74). The function is quite simple and self-explained. I think the problem could occur if the random value is very close to 1.0. The comparing sign '<' on line 82 maybe should be changed to '<=' .

#### Version Information

1.0.0-beta7

**Comment by saudet** [comment]
That should be fixed with 1.0.0-SNAPSHOT.

**Comment by saudet** [comment]
Duplicate of #8800
